### ISSUE TITLE
Contribution

### ISSUE THREAD

**Issue by GouravSardana** [facebook/FAI-PEP#124]
Sorry for being creating an issue in this repo. Are you guys working on Django ?

**Comment by sf-wind** [comment]
No. At least not yet. Do you want to contribute?

**Comment by GouravSardana** [comment]
yup, I am a Backend Developer but yeah i can learn more stuff also.

**Comment by sf-wind** [comment]
That's cool. We can have a meeting to discuss how you can contribute.

**Comment by GouravSardana** [comment]
Okay sure!
Where is the meeting and how can i attend? @sf-wind
